### PR TITLE
Add blockleben

### PR DESCRIPTION
**Pull request by Jxelsw**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/blockleben/background.png
Comment (DoJapHD) [inline minecraft_servers/blockleben/background.png:1]: Compress image

File: minecraft_servers/blockleben/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/blockleben/background@2x.png:1]: Compress image

File: minecraft_servers/blockleben/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "nice_name": "BlockLeben",
+  "direct_ip": "blockleben.de",
+  "server_wildcards": [
+    "blockleben.net"
+  ],
+  "supported_languages": [
+      "de"
+  ],
+  "social": {
+    "web": "https://blockleben.de",
```
Comment (DoJapHD) [inline minecraft_servers/blockleben/manifest.json:12]: Website is file upload?

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/blockleben/manifest.json (hunk 1 above)
Comment (Jxelsw) [reply minecraft_servers/blockleben/manifest.json:12]: Our Website is in maintenance
must there be a website?

Comment (DoJapHD) [reply minecraft_servers/blockleben/manifest.json:12]: If web is still in maintenance, please remove it; you can add it once there's a website

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/blockleben/manifest.json (hunk 1 above)
Comment (Jxelsw) [reply minecraft_servers/blockleben/manifest.json:12]: i cant create a pull request

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**